Example/App.js

```diff
@@ -51,36 +47,33 @@ class MainScreen extends React.Component {
         renderItem={(props) => (
           <MainScreenItem
             {...props}
-            onPressItem={({ key }) => this.props.navigation.navigate(key)}
+            onPressItem={({ key }) => navigation.navigate(key)}
           />
         )}
       />
     );
-  }
 }
 
 const ItemSeparator = () => <View style={styles.separator} />;
 
-class MainScreenItem extends React.Component {
-  _onPress = () => this.props.onPressItem(this.props.item);
-  render() {
-    const { key } = this.props.item;
+const MainScreenItem = (props) => {
+  _onPress = () => props.onPressItem(props.item);
+    const { key } = props.item;
     return (
-      <TouchableHighlight onPress={this._onPress}>
+      <TouchableHighlight onPress={_onPress}>
         <View style={styles.button}>
           <Text style={styles.buttonText}>{SCREENS[key].title || key}</Text>
         </View>
       </TouchableHighlight>
     );
-  }
 }
 
 const MainScreenStack = createStackNavigator();
 
 const ExampleApp = () => (
   <NavigationContainer>
     <MainScreenStack.Navigator>
-      <MainScreenStack.Screen name="Main" component={MainScreen} />
+      <MainScreenStack.Screen name="📱 React Native Screens Examples" component={MainScreen} />
```

Comment:
I would leave the name as "Main" and add the title name in `options` of the screen. It would be weird to use the `navigate` option to a screen with such a name. 

```suggestion
      <MainScreenStack.Screen name="Main" component={MainScreen} options={{title: '📱 React Native Screens Examples'}} />
```

Reply:
yep it's weird not use the name for the screen